test/support/video_compositor_helpers.ex

```diff
@@ -4,37 +4,62 @@ defmodule Membrane.VideoCompositor.Test.Utility do
 
```

Comment:
Maybe we should rename file to the `test_utility.ex` or something similar (to match the module name)?